planex/makesrpm.py

```diff
@@ -0,0 +1,145 @@
+#!/usr/bin/python
+
+"""
+planex-make-srpm: Wrapper around rpmbuild
+"""
+
+import sys
+import subprocess
+import os
+from shutil import copyfile, rmtree
+import fileinput
+import tarfile
+import tempfile
+
+import argparse
+import argcomplete
+from planex.util import add_common_parser_options
+
+
+def parse_args_or_exit(argv):
+    """
+    Parse command line options
+    """
+    parser = argparse.ArgumentParser(description='Planex SRPM building')
+    add_common_parser_options(parser)
+    parser.add_argument(
+        "--topdir", metavar="DIR", default=None,
+        help='Set rpmbuild toplevel directory')
+    parser.add_argument(
+        "--dist", metavar="DIST", default="",
+        help="distribution tag (used in RPM filenames)")
+    argcomplete.autocomplete(parser)
+    return parser.parse_known_args(argv)
+
+
+def parse_patchseries(series_file):
+    """
+    Parse series file and return the list of patches
+    """
+    patches = []
+    for line in open(series_file):
+        line = line.strip()
+        if not line.startswith("#"):
+            patches.append(line)
+
+    return patches
+
+
+def setup_tmp_area():
+    """
+    Create temporary wotking area
+    """
+    tmp_dirpath = tempfile.mkdtemp() + '/'
+    tmp_specs = tmp_dirpath + 'SPECS/'
```

Comment:
Please use os.path.join(tmp_dirpath, "SPECS")
If you do that, you also don't need to append '/' to tmp_dirpath